lib/content-services/src/lib/search/components/search-date-range/search-date-range.component.spec.ts

```diff
@@ -62,39 +65,39 @@ describe('SearchDateRangeComponent', () => {
         expect(component.form).toBeDefined();
     });
 
-    it('should setup the format of the date from configuration', () => {
-        component.settings = { field: 'cm:created', dateFormat: dateFormatFixture };
+    it('should check the format of the date from component', () => {
+        component.settings = { field: 'cm:created', dateFormat: DateFnsUtils.convertMomentToDateFnsFormat(dateFormatFixture) };
         fixture.detectChanges();
 
-        expect(adapter.overrideDisplayFormat).toBe(dateFormatFixture);
+        expect(component.datePickerFormat).toBe(DateFnsUtils.convertMomentToDateFnsFormat(dateFormatFixture));
     });
 
     it('should setup form control with formatted valid date on change', () => {
-        component.settings = { field: 'cm:created', dateFormat: dateFormatFixture };
+        component.settings = { field: 'cm:created', dateFormat: DateFnsUtils.convertMomentToDateFnsFormat(dateFormatFixture) };
         fixture.detectChanges();
 
         const inputString = '20-feb-18';
-        const momentFromInput = moment(inputString, dateFormatFixture);
+        const dateFromInput = parse(inputString, DateFnsUtils.convertMomentToDateFnsFormat(dateFormatFixture), new Date());
 
-        expect(momentFromInput.isValid()).toBeTruthy();
+        expect(isValid(dateFromInput)).toBeTruthy();
 
         component.onChangedHandler({ value: inputString }, component.from);
 
-        expect(component.from.value.toString()).toEqual(momentFromInput.toString());
+        expect(component.from.value.toString()).toEqual(dateFromInput.toString());
     });
 
     it('should NOT setup form control with invalid date on change', () => {
-        component.settings = { field: 'cm:created', dateFormat: dateFormatFixture };
+        component.settings = { field: 'cm:created', dateFormat: DateFnsUtils.convertMomentToDateFnsFormat(dateFormatFixture) };
         fixture.detectChanges();
 
         const inputString = '20.f.18';
-        const momentFromInput = moment(inputString, dateFormatFixture);
+        const dateFromInput = parse(inputString, DateFnsUtils.convertMomentToDateFnsFormat(dateFormatFixture), new Date());
```

Comment:
Date util already has `parseDate` that does this: https://github.com/Alfresco/alfresco-ng2-components/blob/ea657566606dac4b6096cdb4b7c78734fe6db8a4/lib/core/src/lib/common/utils/date-fns-utils.ts#L156

Reply:
Done